acs_registry/protocols/station_query_protocol.py

```diff
@@ -13,24 +15,34 @@
 async def station_query(ctx: Context, sender: str, request: StationQueryRequest):
     ctx.logger.info(f"Car: {sender} requested: {request}")
 
-    stations: list[Station] = ctx.storage.get("stations")
-    stations = filter_stations(stations, request)
+    stations_list = filter_stations(station_collection, request)
 
-    await ctx.send(sender, StationQueryResponse(stations=stations))
+    await ctx.send(sender, StationQueryResponse(stations=stations_list))
 
 
-def filter_stations(stations: list[Station], request: StationQueryRequest) -> list[str]:
-    def is_in_range(station: Station):
-        lat_dif = station[0] - request.lat
-        long_dif = station[1] - request.long
-        distance_square = lat_dif**2 + long_dif**2
+def filter_stations(collection, request: StationQueryRequest) -> list[str]:
```

Comment:
nit: should be renamed to something like `get_stations`

Reply:
renamed